Magento/ruleset.xml

```diff
@@ -72,6 +72,9 @@
     <rule ref="Magento.Exceptions.Namespace">
         <severity>8</severity>
     </rule>
+    <rule ref="Magento.Functions.ThrowCatch">
```

Comment:
Could you please add `<type>warning</type>`. We decided to be as explicit as possible in the ruleset :)